lib/services/common/securityServiceOAuth2.js

```diff
@@ -70,8 +70,7 @@ function auth(trust, callback) {
         body: formData
     };
 
-    logger.debug(context, 'Authentication on the OAuth2 provider [%s] using trust [%s]', url, trust);
-    logger.debug(context, 'Using the following request:\n\n%s\n\n', JSON.stringify(options, null, 4));
+    logger.debug(context, 'Authentication on the OAuth2 provider [%s]', url);
```

Comment:
I understand the change from `'Authentication on the OAuth2 provider [%s] using trust [%s]', url, trust` to `Authentication on the OAuth2 provider [%s]', url` to avoid printing the token. However, what about the removal of `'Using the following request:\n\n%s\n\n', JSON.stringify(options, null, 4)`. Maybe it is due the request may include also sensible information?

Reply:
That's the point @fgalan, if we print the `options`, sensible information may be leaked, so I prefered to remove this log.

Reply:
NTC